i18n/src/commonMain/moko-resources/base/strings.xml

```diff
@@ -626,6 +626,8 @@
     <string name="pref_verbose_logging">Verbose logging</string>
     <string name="pref_verbose_logging_summary">Print verbose logs to system log (reduces app performance)</string>
     <string name="pref_debug_info">Debug info</string>
+    <string name="pref_update_library_manga_titles">Allow updating title for library entries</string>
+    <string name="pref_update_library_manga_titles_summary">If a source has a different title for an entry than the one stored in the library, update it to match the source\'s title.\nWarning: if a manga is renamed, it will be removed from the download queue (if present).</string>
```

Comment:
```suggestion
    <string name="pref_update_library_manga_titles">Update library manga titles to match source</string>
    <string name="pref_update_library_manga_titles_summary">Warning: if a manga is renamed, it will be removed from the download queue (if present).</string>
```